source/_components/climate.intesishome.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: "IntesisHome AC Controller"
+description: "Instructions how to setup IntesisHome integration with Home Assistant"
+date: 2017-07-03 21:38
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: intesishome.png
+ha_category: Climate
+ha_release: 0.49
+ha_iot_class: "Cloud Push"
+---
+
+The `IntesisHome` climate platform let you control [IntesisHome](https://www.intesishome.com) devices. IntesisHome provide integrations with air conditioners including including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+climate:
+  - platform: intesishome
+    username: 'YOUR_USERNAME'
+    password: 'YOUR_PASSWORD'
+```
+
```

Comment:
Definition of configuration variables are missing.

Have a look at another component for examples on how to document the variables.